src/doc/rustc/src/platform-support/x86_64-pc-cygwin.md

```diff
@@ -0,0 +1,44 @@
+# `x86_64-pc-cygwin`
+
+**Tier: 3**
+
+Windows targets supporting Cygwin and MSYS2.
+The `*-cygwin-*` targets are **not** intended as native targets for applications,
+a developer writing Windows applications should use the `*-pc-windows-*` targets instead, which are *native* Windows.
+
+Cygwin is only intended as an emulation layer for UNIX-only programs which do not support the native Windows targets.
```

Comment:
UNIX is the trademark, but we do not use Unix in the sense of "conformant precisely to the Single Unix Specification".
```suggestion
Cygwin is only intended as an emulation layer for Unix-only programs which do not support the native Windows targets.
```